spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+# RSpec.describe UsersController, type: :controller do
+#
+#   describe "GET #show" do
+#     it "returns http success" do
```

Comment:
if you need to not run a test, mark it as pending
https://relishapp.com/rspec/rspec-core/v/3-6/docs/pending-and-skipped-examples/pending-examples